### PR TITLE
V0.1.2

### PR DESCRIPTION
**Pull request by kohei-miyazawa**
## Todo
- I newly added abs, sign function and corresponding specs.

## Reference
- [ABS function](https://dekiru.net/article/4786/)
- [SIGN function](https://dekiru.net/article/4787/)

### CODE REVIEW

**Comment by yiyenene** [comment]
@kohei-miyazawa Thank you for create PR and I apologize for my preparing is not enough.
I created `develop` branch. Could you change this PR to the following?
- Change the target branch to `develop`
- Remove the commit changes version up  [232f5d5](https://github.com/moneyforward/xls_function/pull/5/commits/232f5d5228846fc9857f7a418426c0f21a7efc3a)
    - I will handle the release later.

**Comment by kohei-miyazawa** [comment]
@yiyenene 
Thank you for chacking.
The following two points have been addressed.
・The target branch has been changed to develop.
・Deleted version up commit.

Please check the following.

File: lib/xls_function/locales/ja.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ja:
   xls_function:
     descriptions:
+      abs: ABS(数値) 数値の絶対値を求めます。
```
Comment (yiyenene) [inline lib/xls_function/locales/ja.yml:4]:
```suggestion
      abs: ABS(数値) 数値の絶対値を返します。
```

File: lib/xls_function/locales/en.yml [hunk 2]
```diff
@@ -42,6 +43,7 @@ en:
       roundup: ROUNDUP(number, num_digits) Rounds a number up, away from 0 (zero).
       round: ROUND(number, num_digits) Rounds a number to a specified number of digits.
       second: SECOND(serial_number) Returns the seconds of a time value. The second is given as an integer in the range 0 (zero) to 59.
+      sign: SIGN(number) Returns 1 if the number is positive, -1 if it is negative and 0 if it is zero.
```
Comment (yiyenene) [inline lib/xls_function/locales/en.yml:46]:
```suggestion
      sign: SIGN(number) Determines the sign of a number. Returns 1 if the number is positive, zero (0) if the number is 0, and -1 if the number is negative.
```

File: lib/xls_function/locales/ja.yml [hunk 3]
```diff
@@ -42,6 +43,7 @@ ja:
       roundup: ROUNDUP(数値, 桁数) 数値を指定された桁数に切り上げます。
       round: ROUND(数値, 桁数) 数値を四捨五入して指定された桁数にします。
       second: SECOND(シリアル値) 時刻の秒を返します。 戻り値は 0 (秒) ～ 59 (秒) の範囲の整数となります。
+      sign: SIGN(数値) 数値が正の数であれば1、負の数であれば-1、0であれば0を返します。
```
Comment (yiyenene) [inline lib/xls_function/locales/ja.yml:46]:
```suggestion
      sign: SIGN(数値) 数値の正負を調べます。 戻り値は、数値が正の数のときは 1、0 のときは 0、負の数のときは -1 となります。
```

**Review by yiyenene — CHANGES_REQUESTED** [review]
@kohei-miyazawa I refered to the words at the support site.
Implementation is fine! Thank you.

**Comment by kohei-miyazawa** [comment]
@yiyenene Thank you for chacking.
I corrected for description. Could you check the following?

**Review by yiyenene — APPROVED** [review]
Thanks！

**Pull request merged**